lib/puppet/face/config.rb

```diff
@@ -75,7 +75,7 @@
         if args.length == 1
           puts values.interpolate(args[0].to_sym)
         else
-          args.each do |setting_name|
+          args.sort_by { |key| key }.each do |setting_name|
```

Comment:
Why `sort_by` vs `sort`?

Reply:
That should just be `sort`. Originaly I was trying to sort the key entries, but later changed it to just sort the keys. Let me fix that.